server.py

```diff
@@ -280,7 +282,12 @@ class LastUpdateHandler(BaseHandler):
     """
     @gen.coroutine
     def get(self):
-        body = json.loads(self.request.body)
+        try:
+            body = json.loads(self.request.body)
+        except JSONDecodeError:
+            self.set_status(500)
+            self.write({"Error": "request body is missing."})
```

Comment:
This should return 400 Bad Request since the issue is a malformed or missing body data when this exception is thrown. Also the error message should reflect the fact that the body can just be malformed

Reply:
Good point, fixed.